components/style/values/specified/image.rs

```diff
@@ -31,6 +33,8 @@ use values::generics::position::Position as GenericPosition;
 use values::specified::{Angle, Color, Length, LengthOrPercentage};
 use values::specified::{Number, NumberOrPercentage, Percentage, RGBAColor};
 use values::specified::position::{LegacyPosition, Position, PositionComponent, Side, X, Y};
+#[cfg(feature = "gecko")]
+use values::specified::transform::OriginComponent;
```

Comment:
Let's move this to where it's used, so we don't need add more `cfg`s? (your call really, not a big deal)